locale/en_GB/LC_MESSAGES/django.po

```diff
@@ -5318,646 +5327,3 @@ msgid ""
 msgstr ""
 "You are about to use your %(provider_name)s account to login to "
 "%(site_name)s. As a final step, please complete the following form:"
-
-#~ msgid "Moderator feedback"
-#~ msgstr "Moderator feedback"
-
-#~ msgid "Official Statement"
-#~ msgstr "Official Statement"
-
-#~ msgid "Open"
-#~ msgstr "Open"
-
-#~ msgid "Feedback on the contribution (public)"
-#~ msgstr "Feedback on the contribution (public)"
-
-#~ msgid ""
-#~ "The official feedback will appear below the idea, indicating your "
-#~ "organisation. The idea provider receives a notification."
-#~ msgstr ""
-#~ "The official feedback will appear below the idea, indicating your "
-#~ "organisation. The idea provider receives a notification."
-
-#~ msgid "Processing status (public)"
-#~ msgstr "Processing status (public)"
-
-#~ msgid ""
-#~ "The editing status appears below the title of the idea in red, yellow or "
-#~ "green. The idea provider receives a notification."
-#~ msgstr ""
-#~ "The editing status appears below the title of the idea in red, yellow or "
-#~ "green. The idea provider receives a notification."
-
-#~ msgid "Proposal is archived"
-#~ msgstr "Proposal is archived"
-
-#~ msgid "give feedback"
-#~ msgstr "give feedback"
-
-#~ msgid "Official feedback"
-#~ msgstr "Official feedback"
-
-#~ msgid "Processing status"
-#~ msgstr "Processing status"
-
-#~ msgid "Checked"
-#~ msgstr "Checked"
-
-#~ msgid ""
-#~ "If you define labels, participants can assign their ideas accordingly "
```

Comment:
I can't find those anywhere now, which is weird? Maybe my translations are broken?